lib/tests/api.rs

```diff
@@ -158,27 +158,6 @@ mod api_integration {
 		include!("api/backup.rs");
 	}
 
-	#[cfg(feature = "kv-rocksdb")]
-	mod file {
-		use super::*;
-		use surrealdb::engine::local::Db;
-		use surrealdb::engine::local::File;
-
-		async fn new_db() -> Surreal<Db> {
-			let path = format!("/tmp/{}.db", Ulid::new());
-			let root = Root {
-				username: ROOT_USER,
-				password: ROOT_PASS,
-			};
-			let db = Surreal::new::<File>((path.as_str(), root)).await.unwrap();
-			db.signin(root).await.unwrap();
-			db
-		}
-
-		include!("api/mod.rs");
-		include!("api/backup.rs");
-	}
-
```

Comment:
Let's not remove this as eventually, with our own storage engine, RocksDB will not be the default...